### PR TITLE
Replace build and config with current used version, add ability to mo…

### PR DESCRIPTION
**Pull request by inekrasov-intermedia**
Replace build and config with current used version, add ability to mount input and output

NOTE:
To publish docker container we need intermedia token from docker hub

### CODE REVIEW

File: config.conf [hunk 1]
```diff
@@ -143,7 +125,7 @@ OPUS_BUILD_OUT_PATH="$BUILD_DIR/opus-build-output"
 
 # Belledonne Communication g729 Version to download.
 # If a version is not specified it will download the current master
-BCG729_VERSION=1.1.1
+BCG729_VERSION=master
```
Comment (anikitin-intermedia) [inline config.conf:128]: Why master? I think it is better to use specific version

File: config.conf [hunk 2]
```diff
@@ -199,38 +181,24 @@ DOWNLOAD_SDK=1
...
-DOWNLOAD_OPENH264=1
-# Compiles PJSIP with OpenH264 support; download and build are required to enable it
-ENABLE_OPENH264=1
+# Downloads a fresh copy of OpenH264. Any existing files will be deleted
+DOWNLOAD_OPENH264=0
 
 # On recent versions of OpenH264 building encdemo and decdemo build fails
 # [OpenH264 2.1.x && NDK >= r20b]
-SKIP_OPENH264_DEMO=1
+SKIP_OPENH264_DEMO=0
```
Comment (anikitin-intermedia) [inline config.conf]: Why we need this demo?

**Review by anikitin-intermedia — CHANGES_REQUESTED** [review]

File: config.conf (hunk 1 above)
Comment (inekrasov-intermedia) [reply config.conf:128]: I used our local version and it contains master version. But maybe it was my mistake, can you check it?

File: config.conf (hunk 2 above)
Comment (inekrasov-intermedia) [reply config.conf]: It's the same, but yes, we can disable it, I think.

File: config.conf (hunk 1 above)
Comment (anikitin-intermedia) [reply config.conf:128]: Ok, got it!

File: config.conf (hunk 2 above)
Comment (anikitin-intermedia) [reply config.conf]: No, I don't see this flag in my config.conf file at all

Comment (inekrasov-intermedia) [reply config.conf]: Fixed and returned to 1 (skip)

**Review by anikitin-intermedia — APPROVED** [review]

**Review by dshamaev-intermedia — APPROVED** [review]

**Pull request merged**